browse/controllers/archive_page/__init__.py

```diff
@@ -4,7 +4,13 @@
 from typing import Any, Dict, List, Tuple, Optional
 from http import HTTPStatus as status
 
-from arxiv.taxonomy.definitions import ARCHIVES, ARCHIVES_SUBSUMED, CATEGORIES
+from arxiv.taxonomy.definitions import (
+    ARCHIVES, 
+    ARCHIVES_SUBSUMED, 
+    CATEGORIES,
+    tCategory,
+    tArchive
```

Comment:
nitpick- tArchive and tCategory names feel confusing

Reply:
Suggestions? They're TypedDicts